### PR TITLE
Issue 197

### PR DESCRIPTION
**Pull request by wangsnuxiao**

### CODE REVIEW

File: creation/views.py [hunk 1]
```diff
@@ -135,7 +137,12 @@ def addVenue(request, day):
     yelp_id = request.GET.get("yelp_id")
     venue, created = Venue.objects.get_or_create(yelp_id=yelp_id)
     DayVenue.objects.create(day=day, venue=venue, pos=day.dayvenue_set.count() + 1)
-    rootpath = request.path.split("/searchpage")[0]
+    rootpath = request.path.split("?")[0]
+    # Create Success message
+    request.session["Success_Message"] = "Added %s into Day: %s" % (
+        venue.yelp_id,
```
Comment (yp1383) [inline creation/views.py:143]: It is awesome to show a message when adding a new day venue to a day.
But is it better if we remove the yelp_id from the message?
The user doesn't understand what that means.

Comment (wangsnuxiao) [reply creation/views.py:143]: Sure! The reason I used yelp_id is that I cannot get the name of the Venue without fetching the venue. Remove it is definitely a reasonable option.

File: authentication/views.py [hunk 2]
```diff
@@ -12,39 +12,49 @@
...
     if request.method == "GET":
-        return render(request, "authentication/signup.html")
+        return render(request, "authentication/signup.html", context)
     elif request.method == "POST":
         email = request.POST["email"]
         username = request.POST["username"]
         password1 = request.POST["password1"]
         password2 = request.POST["password2"]
         if password1 != password2:
-            return HttpResponse("two password must align with eachother")
```
Comment (yp1383) [inline authentication/views.py:23]: typo 
two passwords must align with each other

**Review by azraf-a — APPROVED** [review]
Good stuff!

File: creation/views.py [hunk 3]
```diff
@@ -135,7 +137,9 @@ def addVenue(request, day):
     yelp_id = request.GET.get("yelp_id")
     venue, created = Venue.objects.get_or_create(yelp_id=yelp_id)
     DayVenue.objects.create(day=day, venue=venue, pos=day.dayvenue_set.count() + 1)
-    rootpath = request.path.split("/searchpage")[0]
+    rootpath = request.path.split("?")[0]
+    # Create Success message
+    request.session["Success_Message"] = "Added Venue into Day: %s" % (day.id)
```
Comment (sballan) [inline creation/views.py:142]: I don't think the user really cares about the Day ID.  Maybe consider something like this:

`"Added Venue to %s" % (day.name)`

**Review by sballan — COMMENTED** [review]
left a couple of comments

**Review by yp1383 — APPROVED** [review]

**Review by sballan — COMMENTED** [review]
Looks great!

**Pull request merged**